### PR TITLE
docs: added AI Studio to the list of UIs [no ci]

### PR DESCRIPTION
**Pull request by SommerEngineering**
- [x] I have read the [contributing guidelines](https://github.com/ggerganov/llama.cpp/blob/master/CONTRIBUTING.md)
- Self-reported review complexity:
  - [x] Low
  - [ ] Medium
  - [ ] High

### CODE REVIEW

**Comment by SommerEngineering** [comment]
I added [AI Studio](https://github.com/MindWorkAI/AI-Studio) to the list of compatible UIs.

**Comment by SommerEngineering** [comment]
Do I need to do anything else as the author to get my PR accepted? Or do I just need to wait for approval? 😀

**Review by ggerganov — APPROVED** [review]

**Pull request merged**
